gtsam/slam/dataset.h

```diff
@@ -76,14 +76,24 @@ enum KernelFunctionType {
 
 /// Return type for auxiliary functions
 typedef std::pair<Key, Pose2> IndexedPose;
+typedef std::pair<Key, Point2> IndexedLandmark;
 typedef std::pair<std::pair<Key, Key>, Pose2> IndexedEdge;
 
 /**
  * Parse TORO/G2O vertex "id x y yaw"
  * @param is input stream
  * @param tag string parsed from input stream, will only parse if vertex type
  */
-GTSAM_EXPORT boost::optional<IndexedPose> parseVertex(std::istream& is,
+GTSAM_EXPORT boost::optional<IndexedPose> parseVertexPose(std::istream& is,
```

Comment:
I see no reason for an API breaking name change, and if it were to be implemented we need a deprecated section with the old name.

Reply:
Sounds good. I'll use the compiler definition from #422 if that's okay?

Reply:
I prefer *not* to do the name change